socorro/processor/rules/breakpad.py

```diff
@@ -70,6 +70,70 @@ def action(self, raw_crash, dumps, processed_crash, status):
         )
 
 
+class TruncateStacksRule(Rule):
+    """Truncate stacks that are too large
+
+    This rule truncates stacks that are greater than MAX_FRAMES. It truncates the stacks
+    in the middle with the thinking that the beginning and end of the stack are the most
+    interesting.
+
+    """
+
+    MAX_FRAMES = 500
```

Comment:
We picked 500 because it's enough to see what's going on, but not so much that the webapp is overwhelmed. We can change this number if it's too much/little.